### PR TITLE
[14.0.0][MIG] account_template_active

### PR DESCRIPTION
**Pull request by StefanRijnhart**
Port of https://github.com/OCA/account-financial-tools/pull/978

### CODE REVIEW

File: account_template_active/readme/USAGE.rst
Comment (legalsylvain) [inline account_template_active/readme/USAGE.rst]: I think this part is obsolete, as you introduce a dependency to ``account_menu`` in the manifest.

**Review by legalsylvain — COMMENTED** [review]
One minor remark. LGTM otherwise. Thanks for porting this module.

**Review by legalsylvain — APPROVED** [review]

File: account_template_active/readme/USAGE.rst
Comment (StefanRijnhart) [reply account_template_active/readme/USAGE.rst]: Ah thanks! Removed.

**Comment by legalsylvain** [comment]
hi @OCA/accounting-maintainers : some could review and merge this PR ? 

the diff review is quite simple : https://github.com/OCA/account-financial-tools/pull/1187/commits/73fe7cbcad5d8b6eb12afa07ed06ab66e49000b8

thanks !

**Review by jcdrubay — COMMENTED** [review]
LGTM after checking the last commit only.

Just wondering why not using the `widget="boolean_toggle" ` on both form views

- view_account_template_form
- view_account_tax_template_form

**Comment by StefanRijnhart** [comment]
@jcdrubay The boolean fields that there are on these forms (in core Odoo) do not apply the widget either (not in Odoo/master either). Mixing boolean widget looks ugly IMHO, and the usability improvement of the widget is less obvious in the case of the form view. Unless you've noticed that the widget is applied on form views throughout Odoo (14, or master), we have to assume that the convention in core Odoo is not to apply the widget in form views and we should stick to that.

File: account_template_active/views/view_account_tax_template.xml
Comment (jcdrubay) [inline account_template_active/views/view_account_tax_template.xml]:
```suggestion
                <field name="active" />
```

Comment (StefanRijnhart) [reply account_template_active/views/view_account_tax_template.xml]: Thanks, taken!

**Review by jcdrubay — APPROVED** [review]

**Comment by legalsylvain** [comment]
@jcdrubay : thanks for your review.

**Review by joao-p-marques — COMMENTED** [review]
Code review 👍 

Could you just squash the 2 latest commits together please so that they follow the standard?

**Comment by StefanRijnhart** [comment]
@joao-p-marques squash done, thanks!

**Review by joao-p-marques — APPROVED** [review]
Great! I think we can merge this, then

**Pull request merged**
